pkg/platform/runtime/buildscript/buildexpression.go

```diff
@@ -28,16 +29,17 @@ func NewScriptFromBuildExpression(expr []byte) (*Script, error) {
 	if !ok {
 		return nil, errs.New("'let' key is not a JSON object")
 	}
+	inValue, ok := letMap["in"]
+	if !ok {
+		return nil, errs.New("Build expression's 'let' object has no 'in' key")
+	}
+	delete(letMap, "in")
```

Comment:
Prevent duplication of "in" field when writing the build script. This may change depending on how we want the build script syntax to look in the end.

Reply:
This comment is very helpful, so it should probably be in the code.